### DISCUSSION TITLE
Markdown: jump to anchor

### DISCUSSION THREAD

**Discussion by HeinzWuert** [kimai/kimai#4128] [Plugins]
Custom Content: In standard Markdown, place an anchor **a name="abcd"></a** where you want to link to 
Refer to it on the same page by **link text](#abcd)**
I know, that the first sign in the commands is missing.

Link seems to be ok, but the anchor himself not.
Any ideas?

**Comment by kevinpapst** [comment]
Try the demo, seems to be working correctly in 2.0, don't know about 1.x though.
